src/Features/Core/Portable/GenerateMember/GenerateVariable/AbstractGenerateVariableService.GenerateParameterCodeAction.cs

```diff
@@ -27,7 +27,7 @@ public override string Title
                 get
                 {
                     var text = _includeOverridesAndImplementations
-                        ? FeaturesResources.Generate_parameter_0_and_overrides_implementations 
```

Comment:
>  [](start = 94, length = 1)

I'm not sure that it is OK to keep this and other formatting changes in unrelated files in this PR. Please consider moving them out.

Reply:
Those are all in a single commit 0a92a419a7f9e26e71a71eea2e2631279ebc46f4. But I can revert that if you prefer.